src/misc/ontime.ts

```diff
@@ -0,0 +1,156 @@
+import { Bukkit, Statistic } from 'org.bukkit';
+import { CommandSender } from 'org.bukkit.command';
+import { Player } from 'org.bukkit.entity';
+import { PlayerQuitEvent } from 'org.bukkit.event.player';
+import * as yup from 'yup';
+import { DatabaseEntry } from '../common/datas/database';
+import { dataType } from '../common/datas/holder';
+import { dataView } from '../common/datas/view';
+
+const OntimeDatabaseEntry = new DatabaseEntry('ontime', 'ontime-key');
```

Comment:
I'd move this to inside of dataView call. If you don't do that, please use CAMEL_CASE.